llvm/lib/Target/NVPTX/NVPTXIntrinsics.td

```diff
@@ -7413,9 +7436,9 @@ class LDMATRIX<WMMA_REGINFO Frag, bit Transposed, string Space>
 defset list<WMMA_INSTR> LDMATRIXs  = {
   foreach transposed = [false, true] in {
     foreach space = [".shared", ""] in {
-      foreach frag = NVVM_MMA_OPS.all_ldmatrix_ops in
-        if NVVM_LDMATRIX_SUPPORTED<frag>.ret then
-          def : LDMATRIX<WMMA_REGINFO<frag, "ldmatrix">, transposed, space>;
+        foreach frag = NVVM_MMA_OPS.all_ldmatrix_ops in
+          if NVVM_LDMATRIX_SUPPORTED<frag, transposed>.ret then
+            def : LDMATRIX<WMMA_REGINFO<frag, "ldmatrix">, transposed, space>;
```

Comment:
Looks like these lines got shifted by an extra indentation level. You may want to undo that.

Reply:
Thanks @Artem-B! Removed the addtional indentation